csv_export.php

```diff
@@ -68,16 +68,8 @@
 
 $t_filename = csv_get_default_filename();
 
-# Send headers to browser to activate mime loading
```

Comment:
> Does this really add value?

I'm not sure, I was just curious as to why it got removed.

I just went through history, this comment is there since the CSV export feature was added by Ken in 2002... 49659dd8e1d824966a3c8b1c5050677fe3824300, and TBH I don't really see the point either.